build-system/tasks/extension-helpers.js

```diff
@@ -561,18 +561,76 @@ async function buildExtensionJs(path, name, version, latestVersion, options) {
   }
 
   if (name === 'amp-script') {
-    // Copy @ampproject/worker-dom/dist/amp/worker/worker.js to dist/ folder.
-    const dir = 'node_modules/@ampproject/worker-dom/dist/amp/worker/';
-    const file = `dist/v0/amp-script-worker-${version}`;
-    // The "js" output is minified and transpiled to ES5.
-    fs.copyFileSync(dir + 'worker.js', `${file}.js`);
-    // The "mjs" output is unminified ES6 and has debugging flags enabled.
-    fs.copyFileSync(dir + 'worker.mjs', `${file}.max.js`);
-
-    // Same as above but for the nodom worker variant.
-    const noDomFile = `dist/v0/amp-script-worker-nodom-${version}`;
-    fs.copyFileSync(dir + 'worker.nodom.js', `${noDomFile}.js`);
-    fs.copyFileSync(dir + 'worker.nodom.mjs', `${noDomFile}.max.js`);
+    copyWorkerDomResources(version);
+  }
+}
+
+async function copyWorkerDomResources(version) {
+  const workerDomDir = 'node_modules/@ampproject/worker-dom';
+  const targetDir = 'dist/v0';
+
+  log(
+    green('Copying @ampproject/worker-dom resources:'),
+    cyan('worker.js, worker.nodom.js')
+  );
```

Comment:
Sorry about making you change this again, but I pointed you to the wrong logging example earlier.

![image](https://user-images.githubusercontent.com/26553114/92625917-3e741580-f297-11ea-9359-3176381bc091.png)

Could we change this as follows so the step can be timed?

At the start of this function:
```js
  const startTime = Date.now(); 
```

At the end of this function:
```js
    endBuildStep(
      '@ampproject/worker-dom resources',
      'worker.js, worker.nodom.js',
      startTime
    );
```

Reply:
its np. ill make the change. appreciate the thoroughness 👍